entc/gen/type_test.go

```diff
@@ -25,17 +25,17 @@ func TestType(t *testing.T) {
 
 	_, err = NewType(&Config{Package: "entc/gen"}, &load.Schema{
 		Fields: []*load.Field{
-			{Unique: true, Default: true, Info: &field.TypeInfo{Type: field.TypeInt}},
+			{Name: "foo", Unique: true, Default: true, Info: &field.TypeInfo{Type: field.TypeInt}},
 		},
 	})
-	require.Error(err, "unique field can not have default")
+	require.EqualError(err, "unique field \"foo\" cannot have default value")
```

Comment:
I changed the test to `EqualError` because the test was wrong, it just accept the `field name cannot be empty` error